google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/v1/AckResponse.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.pubsub.v1;
+
+public enum AckResponse {
+  PERMISSION_DENIED,
+  FAILED_PRECONDITION,
+  SUCCESSFUL,
+  INVALID
```

Comment:
Another code should be "OTHER" to capture all the other cases (though those should be very rare).